growth/too/tasks/scheduler.py

```diff
@@ -157,8 +162,12 @@ def schedule_ztf(json_data):
                            'validity_window_mjd':
                                json_data["validity_window_mjd"],
                            'queue_type': 'list'})
-    r.raise_for_status()
 
+    if (r.status_code == 200) or (r.status_code == 400):
```

Comment:
Wait a second. Why `danger` if `r.status_code == 200`?

Reply:
This is the case where we resubmitted the same queue twice. It isn't a success.

Reply:
HTTP 200 means "OK".

Reply:
Ok well I am just interpreting what we are getting back from the scheduler. @ebellm